pytd/query_engine.py

```diff
@@ -95,7 +95,8 @@ def execute(self, query, **kwargs):
         cur = self.cursor(**kwargs)
         self.executed = cur.execute(query)
         rows = cur.fetchall()
-        columns = [desc[0] for desc in cur.description]
+        # cur.description is None for CREATE and DROP statements in recent version of Trino
+        columns = [desc[0] for desc in cur.description] if cur.description else []
```

Comment:
Shouldn't we pass along `None` in case `cur.description` is `None`? .i.e `columns` will be `None` instead of `[]` if `cur.description == None`

Reply:
Thanks!
My point was that this NPE is caused by non-null-safe implementation and returning `None` here can cause a new NPE in user-side non-null-safe implementation.
However, it is understandable that returning `None` here is more clear.
I would like to ask your opinion about it 🙏

Reply:
IMHO, the closer the returned value is to the request response, the better. So, if cur.description is ['a', 'b', 'c'], [] or None the columns will be ['a', 'b', 'c'], [] or None respectively. Unless it is prohibitively inconvenience for downstream `pytd` user of course.

Reply:
@tung-vu-td 
Updated, thanks!

Reply:
Thanks @Matts966 for the fix 🙇 Do you need a new release soon?

Reply:
@tung-vu-td 
We already patched it internally but it would be nice to have the new release as this can happen when importing data with `writer='insert_into'`.

Reply:
As a workaround, users can avoid this problem by using `msgpack` format.

Reply:
Thanks for letting us know! We're will try to cut a release together with other fixes then (potientially this https://github.com/treasure-data/pytd/pull/128)